lib/grammers-client/src/types/media.rs

```diff
@@ -267,6 +273,37 @@ impl Sticker {
     }
 }
 
+impl Contact {
+    pub(crate) fn from_media(contact: tl::types::MessageMediaContact) -> Self {
+        Self { contact }
+    }
+
+    pub(crate) fn to_input_media(&self) -> tl::types::InputMediaContact {
+        tl::types::InputMediaContact {
+            phone_number: self.contact.phone_number.clone(),
+            first_name: self.contact.first_name.clone(),
+            last_name: self.contact.last_name.clone(),
+            vcard: self.contact.vcard.clone(),
+        }
+    }
+
+    pub fn phone_number(&self) -> String {
+        self.contact.phone_number.clone()
+    }
```

Comment:
Perhaps we should return a string slice here, instead of cloning it. same below.
```suggestion
    pub fn phone_number(&self) -> &str {
        self.contact.phone_number.as_str()
    }
```

Reply:
Yes, it's better if we return a `&str` here (and the other methods too; the user can clone it if they need the owned version).